cli/cli.ts

```diff
@@ -4674,22 +4687,32 @@ function buildJResSpritesCoreAsync(parsed: commandParser.ParsedCommand) {
 
                 let storeIcon = false
 
-                if (storeIcon) {
+                // These are space-separated lists of tags
+                let tags = `${info.tags || ""} ${metaInfo.tags || ""}`.split(" ").filter(t => !!t);
```

Comment:
👍 To confirm, you're swapping order here so file specific 'info' tags come first in list / override folder's 'meta' tags?

Reply:
Nope, this was an accidental change. Should I revert it?

Reply:
This is a good change, when grabbing tags it will pull off the first value which is more closely associated with the given file in case a given png wants to change a general tag